docs/consume-packages/Central-Package-Management.md

```diff
@@ -98,50 +129,54 @@ Consider the following repository structure:
         └─ 📄 Project2.csproj
 ```
 
-- `Project1.csproj` will load the `Directory.Packages.props` file in the `Repository\Solution1\` directory first and it must manually import any parent ones if desired.
-  ```xml
-  <Project>
-    <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
-    <ItemGroup>
-      <PackageVersion Update="Newtonsoft.Json" Version="12.0.1" />
-    </ItemGroup>
-  </Project>
-  ```
-- `Project2.csproj` will evaluate the `Directory.Packages.props` file in the root directory.
+`Project1.csproj` will use the `Directory.Packages.props` file in the `Repository\Solution1\` directory.
+If you want to include settings from a parent `Directory.Packages.props`, you must manually import it.
 
-**Note:** MSBuild will not automatically import each `Directory.Packages.props` for you, only the first one found in the project directory or any parent directory.  If you have multiple
-`Directory.Packages.props` files, you must import any files in parent directories manually.
+```xml
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
+  <ItemGroup>
+    <PackageVersion Update="Newtonsoft.Json" Version="12.0.1" />
+  </ItemGroup>
+</Project>
+```
 
-## Get started
+`Project2.csproj` will evaluate the `Directory.Packages.props` file in the root directory.
+
+> [!NOTE]
+> MSBuild will only automatically import the first `Directory.Packages.props` file it finds in the project directory or any parent directory.
+> If you have multiple such files, you must manually import parent files as needed.
+
+## Getting Started
 
 To fully onboard your repository, follow these steps:
 
-1. Create a new file at the root of your repository named `Directory.Packages.props` that declares your centrally defined package versions and set
- the MSBuild property `ManagePackageVersionsCentrally` to `true`.
+1. Create a new file at the root of your repository named `Directory.Packages.props` that declares your centrally defined package versions and set the MSBuild property `ManagePackageVersionsCentrally` to `true`.
 2. Declare `<PackageVersion />` items in your `Directory.Packages.props`.
 3. Declare `<PackageReference />` items without `Version` attributes in your project files.
 
-For an idea of how central package management may look like, refer to our [samples repo](https://github.com/NuGet/Samples/tree/main/CentralPackageManagementExample).
+For an example of how Central Package Management may look, refer to our [samples repository](https://github.com/NuGet/Samples/tree/main/CentralPackageManagementExample).
 
-## Transitive pinning
+## Pinning Transitive Packages to Different Versions
```

Comment:
This is a little confusing for me now. 
I liked the previous title much better. 

Transitive Pinning is not about different versions in any way, so I feel like that should be taken out of the title. 

It's also a bit of a keyword as well.

Reply:
I'll revert this.